packages/astro/test/astro-minification-html.test.js

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { loadFixture, isWindows } from './test-utils.js';
+
+describe('minification html', () => {
+	describe('in the dev', () => {
+		let fixture;
+		let devServer;
+		const regex = /[\r\n]+/gm;
+		before(async () => {
+			fixture = await loadFixture({
+				root: './fixtures/minification-html/',
+			});
+			devServer = await fixture.startDevServer();
+		});
+
+		after(async () => {
+			devServer.stop();
+		});
+
+		it('Verify that the HTML code is compressed in the dev', async () => {
+			let res = await fixture.fetch(`/`);
+			expect(res.status).to.equal(200);
+			const html = await res.text();
+			expect(regex.test(html.slice(-100))).to.equal(false);
+		});
+
+	})
+	describe('build', () => {
+		let fixture;
+		const regex = /[\r\n]+/gm;
```

Comment:
nit: no need to repeat the regex, we can define it once at the top of the file, and we can give it a better name